sample/iosApp/iosApp/AppDelegate.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  AppDelegate.swift
+//  iosApp
+//
+//  Created by Kevin Galligan on 1/17/19.
+//  Copyright © 2019 Kevin Galligan. All rights reserved.
```

Comment:
either remove or replace with squares copyright. I think? @JakeWharton

Reply:
Will remove. Looking up xcode defaults. That's autogen stuff

Reply:
Yeah I think you can remove the copyright from defaults but xcode forces the "created by". But I haven't checked in years

Reply:
I have another commit coming with those changes but moving a few things around. A little later today.